tests/unit/sagemaker/jumpstart/test_types.py

```diff
@@ -1193,8 +1193,15 @@ def test_training_configs_parsing():
     assert config.config_components["neuron-training"] == JumpStartConfigComponent(
         "neuron-training",
         {
+            "default_training_instance_type": "ml.trn1.2xlarge",
             "supported_training_instance_types": ["ml.trn1.xlarge", "ml.trn1.2xlarge"],
             "training_artifact_key": "artifacts/meta-textgeneration-llama-2-7b/neuron-training/model/",
+            "training_ecr_specs": {
+                "framework": "huggingface",
+                "framework_version": "2.0.0",
+                "py_version": "py310",
+                "huggingface_transformers_version": "4.28.1",
+            },
```

Comment:
why do we need this?

Reply:
This is to satisfy parsing the default image for `neuron-training` config, otherwise a config component named neuron-training still has a `py3` version in the base spec